packages/app/src/app/components/UserSearchInput/UserSearchInput.tsx

```diff
@@ -11,17 +12,27 @@ type User = {
 
 interface IUserAutoComplete {
   inputValue: string;
+  allowSelf: boolean;
   children: (answer: {
     users: User[];
     loading: boolean;
     error: Error | null;
   }) => JSX.Element;
 }
 
-const UserAutoComplete = ({ inputValue, children }: IUserAutoComplete) => {
+const UserAutoComplete = ({
+  inputValue,
+  children,
+  allowSelf,
+}: IUserAutoComplete) => {
   const [users, setUsers] = React.useState<User[]>([]);
   const [loading, setLoading] = React.useState<boolean>(true);
   const [error, setError] = React.useState<Error | null>(null);
+  const {
+    state: {
+      user: { username },
```

Comment:
I think you should also check if user is there, sometimes people will use this component while not signed in.

Reply:
Fixed 👍